crimethinc/website
# Add PT locale for pt.crimethinc.com

# How does this pull request make you feel (in animated GIF format)?

🎉

# What does this pull request do?

Adds `pt` to list of available locales.
Duplicates the `en` YAML files, adds some PT values.

# How should this be manually tested?

Go to https://pt.crimethinc.com
You should see at least "Books" in PT in the header nav.
Footer nav has some PT bits.
https://pt.crimethinc.com/tools has some intros in PT too.

# Is there any background context you want to provide for reviewers?

Adding another locale subdomain to the site!
